source/includes/apps/capabilities/_delivery_carriers.md

```diff
@@ -1,11 +1,124 @@
 ## Delivery Carriers
 
-TODO
-
-* how to configure in meta.json
-* the rates url is called when selecting a rate in the order screen and the checkout
-  - document the data we send to the rates url (like distance, addresses and products)
-  - it expects rates to be returned in a fixed format
-  - we need to document errors you can return on the rates_url response
-* your app should respond within 3000ms
-* mention scoping
+This capability allows your app to add custom delivery carriers to provide shipping rates for orders.
+
+### Configuration
+
+```shell
+curl --request POST \
+  --url 'https://example.booqable.com/api/4/app_carriers' \
+  --header 'content-type: application/json' \
+  --data '{
+    "data": {
+      "type": "app_carriers",
+      "attributes": {
+        "identifier": "my_delivery_service",
+        "rates_url": "https://my-app.com/api/v1/rates"
+      }
+    }
+  }'
+```
+
+When a user installs your delivery carrier app, you must create a delivery carrier record using Booqable's [Carriers API endpoint](/v4.html#app-carriers). The carrier requires:
+
+* `identifier` - A unique identifier for your carrier.
+* `rates_url` - The URL where Booqable will send rate calculation requests.
+* `app_subscription_id` - The ID of the app subscription (provided during installation).
+
+#### App installation flow
```

Comment:
This is not how the flow works in the Deliveries app, we create the carrier immediately after exchanging the code for the access token as part of the install app step. See: https://github.com/booqable/deliveries/blob/c26f25357794c1a3da50cb0b5e1f893b53cab5f2/app/services/oauth.rb#L23-L26